### PR TITLE
Only ask to update CHANGES.md when merging into master.

### PR DESCRIPTION
**Pull request by OmarShehata**
Resolves https://github.com/AnalyticalGraphicsInc/cesium-concierge/issues/63. Passes all tests and linting.

### CODE REVIEW

File: lib/commentOnOpenedPullRequest.js [hunk 1]
```diff
@@ -80,13 +81,15 @@ commentOnOpenedPullRequest._implementation = function (pullRequestFilesUrl, pull
         });
 };
 
-commentOnOpenedPullRequest._askAboutChanges = function (files) {
+commentOnOpenedPullRequest._askAboutChanges = function (files, baseBranch) {
+    var changesMDWasUpdated = false;
```
Comment (ggetz) [inline lib/commentOnOpenedPullRequest.js]: I think we don't need to run the check to see if `CHANGES.MD` has been updated at all. I think just check `baseBranch === 'master'`, and if so, return false before running the file tests.

File: specs/lib/commentOnOpenedPullRequestSpec.js [hunk 2]
```diff
@@ -56,16 +63,18 @@ describe('commentOnOpenedPullRequest', function () {
...
     it('commentOnOpenedPullRequest._askAboutChanges works', function () {
-        expect(commentOnOpenedPullRequest._askAboutChanges(['CHANGES.md'])).toBe(false);
-
-        expect(commentOnOpenedPullRequest._askAboutChanges([])).toBe(true);
-        expect(commentOnOpenedPullRequest._askAboutChanges(['CHANGES.MD'])).toBe(true);
-        expect(commentOnOpenedPullRequest._askAboutChanges(['leadingCHANGES.md'])).toBe(true);
-        expect(commentOnOpenedPullRequest._askAboutChanges(['CHANGES.mdtrailing'])).toBe(true);
+        expect(commentOnOpenedPullRequest._askAboutChanges(['CHANGES.md'],'master')).toBe(false);
+        expect(commentOnOpenedPullRequest._askAboutChanges(['file.txt'],'feature-branch')).toBe(false);
+        expect(commentOnOpenedPullRequest._askAboutChanges([])).toBe(false);
```
Comment (ggetz) [inline specs/lib/commentOnOpenedPullRequestSpec.js]: Still pass the `baseBranch` here

File: lib/commentOnOpenedPullRequest.js (hunk 1 above)
Comment (OmarShehata) [reply lib/commentOnOpenedPullRequest.js]: Good catch! I updated it.

**Comment by OmarShehata** [comment]
Should be good now. I reverted the test case for when no files were changed back to true. Our statement that "out of all the files changed, if changes.md wasn't changed, remind them to change it" is correct. I don't think we should need to add a special case to say "but if no files were changed don't post your note".

**Comment by ggetz** [comment]
Thanks @OmarShehata! Everything looks good, just checkout that one eslint error and this will be good to merge.

**Comment by OmarShehata** [comment]
Should be all good now! @ggetz

**Comment by ggetz** [comment]
Awesome, thanks @OmarShehata. Do you want to try to deploy and test?

**Comment by OmarShehata** [comment]
It works! See

* https://github.com/AnalyticalGraphicsInc/cesium-concierge/pull/102
* https://github.com/AnalyticalGraphicsInc/cesium-concierge/pull/101

**Pull request merged**